app/static/i18n/de.json

```
@@ -43,7 +43,9 @@
         "AFFECTED_NODES": "Betroffene Knoten",
         "INFO": "INFO",
         "WARNING": "WARNUNG",
-        "CRITICAL": "KRITISCH"
+        "CRITICAL": "KRITISCH",
+        "QUERIES_PER_SECOND": "Anfragen Pro Sekunde",
+        "QUERY_SPEED": "Abfragegeschwindigkeit (ms)"
```

Would this better be called "QUERY_DURATION"? Not totally sure, I would also be reasonably fine with SPEED.
```suggestion
        "QUERY_DURATION": "Abfragedauer (ms)"
```

---

I am indifferent, but would need good translations for EN,DE,FR,ES :D 
I only moved it from the monitoring plugin.

---

Then let's keep it like it was.